fate_flow/manager/queue_manager.py

```diff
@@ -87,8 +87,7 @@ def del_event(self, event):
             if not ret:
                 raise Exception('job not in redis queue')
         except Exception as e:
-            stat_logger.error('delete event from redis queue failed')
-            stat_logger.exception(e)
+            stat_logger.info('delete event from redis queue failed:{}'.format(str(e)))
```

Comment:
What is the purpose of lowering the log level?

Reply:
This exception is thrown when a stop operation is performed while the job is running. The log level is lowered to avoid misleading users.

Reply:
This exception is thrown when a stop operation is performed while the job is running. The log level is lowered to avoid misleading users.